<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPart</class>
 <widget class="QWidget" name="FormPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Партии проволоки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="DateEdit" name="dateEditBeg">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DateEdit" name="dateEditEnd">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdUpdate">
           <property name="toolTip">
            <string>Обновить список партий</string>
           </property>
           <property name="text">
            <string>Обновить</string>
           </property>
           <property name="icon">
            <iconset theme="view-refresh">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TableView" name="tableViewPart">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkBoxOnly">
           <property name="text">
            <string>Только марка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxOnly">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Номер</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Стан</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="comboBoxVol">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxSrcPart">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Марка</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Подтяжка</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="comboBoxIst">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Коммент.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLineEdit" name="lineEditKvo"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Исх. парт.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Волочиль.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="4">
          <widget class="QLineEdit" name="lineEditComment"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxPodt">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Масса, кг</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Источ.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QComboBox" name="comboBoxType">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="toolButtonSrc">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Фильтр исходных партий</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEditNum">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Дата</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DateEdit" name="dateEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="toolButtonPodt">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Фильтр подтяжек</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBoxMar">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Диам.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxDiam">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Намотка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPack">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Варианты намотки партии</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAddPack">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Добавить носитель</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonDelPack">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Удалить носитель</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonEdtPack">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Изменить носитель</string>
           </property>
           <property name="text">
            <string>Ред.</string>
           </property>
           <property name="icon">
            <iconset theme="accessories-text-editor">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPack">
           <property name="text">
            <string>Варианты намотки</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTu">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>Нормативная документация</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="DbViewer" name="tableViewSrcTu">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonCopyTu">
            <property name="toolTip">
             <string>Копировать значения в сертификат</string>
            </property>
            <property name="text">
             <string>-&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DbViewer" name="tableViewPartiTu"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Поступление</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QGroupBox" name="groupBoxNam">
               <property name="minimumSize">
                <size>
                 <width>440</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>Намотка</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="DbViewer" name="tableViewNam"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxAdd">
               <property name="title">
                <string>Упаковка</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="DbViewer" name="tableViewInCex"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QGroupBox" name="groupBoxShip">
               <property name="minimumSize">
                <size>
                 <width>440</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>Отгрузки</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="DbViewer" name="tableViewShip"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxDef">
               <property name="title">
                <string>Списание</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="DbViewer" name="tableViewOutCex"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Испытания</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QGroupBox" name="groupBoxChem">
             <property name="title">
              <string>Испытания</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="DbViewer" name="tableViewMechReal">
                <property name="maximumSize">
                 <size>
                  <width>270</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DbViewer" name="tableViewProvChem">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="cmdCopyChem">
                <property name="toolTip">
                 <string>Копировать значения в сертификат</string>
                </property>
                <property name="text">
                 <string>-&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DbViewer" name="tableViewWirePartChem"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxMech">
             <property name="title">
              <string>Механика сертификат</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="DbViewer" name="tableViewWirePartMech"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutData">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cmdLblSpool">
           <property name="toolTip">
            <string>Этикетка для носителя</string>
           </property>
           <property name="text">
            <string>Этикетка (носитель)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdLblPack">
           <property name="toolTip">
            <string>Этикетка для упаковки</string>
           </property>
           <property name="text">
            <string>Этикетка (упаковка)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cmdSert">
           <property name="toolTip">
            <string>Сертификат на партию</string>
           </property>
           <property name="text">
            <string>Сертификат</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>olap/tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditNum</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>comboBoxIst</tabstop>
  <tabstop>comboBoxSrcPart</tabstop>
  <tabstop>comboBoxVol</tabstop>
  <tabstop>comboBoxPodt</tabstop>
  <tabstop>comboBoxType</tabstop>
  <tabstop>comboBoxMar</tabstop>
  <tabstop>comboBoxDiam</tabstop>
  <tabstop>lineEditKvo</tabstop>
  <tabstop>lineEditComment</tabstop>
  <tabstop>comboBoxPack</tabstop>
  <tabstop>toolButtonAddPack</tabstop>
  <tabstop>toolButtonDelPack</tabstop>
  <tabstop>toolButtonEdtPack</tabstop>
  <tabstop>tableViewNam</tabstop>
  <tabstop>tableViewInCex</tabstop>
  <tabstop>tableViewShip</tabstop>
  <tabstop>tableViewOutCex</tabstop>
  <tabstop>tableViewMechReal</tabstop>
  <tabstop>tableViewProvChem</tabstop>
  <tabstop>cmdCopyChem</tabstop>
  <tabstop>tableViewWirePartChem</tabstop>
  <tabstop>tableViewWirePartMech</tabstop>
  <tabstop>cmdLblSpool</tabstop>
  <tabstop>cmdLblPack</tabstop>
  <tabstop>cmdSert</tabstop>
  <tabstop>dateEditBeg</tabstop>
  <tabstop>dateEditEnd</tabstop>
  <tabstop>cmdUpdate</tabstop>
  <tabstop>tableViewPart</tabstop>
  <tabstop>checkBoxOnly</tabstop>
  <tabstop>comboBoxOnly</tabstop>
  <tabstop>toolButtonSrc</tabstop>
  <tabstop>toolButtonPodt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
